tensorflow/python/training/evaluation.py

```diff
@@ -91,6 +91,8 @@ def __init__(self, num_evals, log_progress=True):
     self._num_evals = num_evals
     self._evals_completed = None
     self._log_progress = log_progress
+    # Log only every 10th evaluation if more than 100 total.
+    self._log_frequency = 1 if num_evals < 100 else 10
```

Comment:
Two small changes:

1) num_evals could be None, so you have a if condition check here otherwise num_evals < 100 will trigger TypeError in python3

2) Probably I did not express clearly sorry. I meant 1/10 of the progress, so, I was thinking

math.floor(num_evals / 10.)